moveit_ros/planning/trajectory_execution_manager/test/test_execution_manager.cpp

```diff
@@ -1,107 +1,103 @@
+  moveit_msgs::RobotTrajectory traj1;
+  moveit_msgs::RobotTrajectory traj2;
+};
 
-  std::cout << "4:\n";
-  if (!tem.ensureActiveControllersForJoints(std::vector<std::string>(1, "lj1")))
-    ROS_ERROR("Fail!");
+TEST_F(MoveItCppTest, EnsureActiveControllersForJointsTest)
+{
+  ASSERT_TRUE(
+      trajectory_execution_manager_ptr->ensureActiveControllersForJoints(std::vector<std::string>(1, "panda_joint1")));
+}
 
-  std::cout << "5:\n";
-  if (!tem.ensureActiveController("left_arm_head"))
-    ROS_ERROR("Fail!");
-  std::cout << "6:\n";
-  if (!tem.ensureActiveController("arms"))
-    ROS_ERROR("Fail!");
+TEST_F(MoveItCppTest, ensureActiveControllerTest)
+{
+  ASSERT_TRUE(trajectory_execution_manager_ptr->ensureActiveController("fake_panda_arm_controller"));
+}
 
+TEST_F(MoveItCppTest, ExecuteEmptySetOfTrajectoriesTest)
+{
   // execute with empty set of trajectories
-  tem.execute();
-  if (!tem.waitForExecution())
-    ROS_ERROR("Fail!");
+  trajectory_execution_manager_ptr->execute();
+  auto last_execution_status = trajectory_execution_manager_ptr->waitForExecution();
+  ASSERT_TRUE(last_execution_status == moveit_controller_manager::ExecutionStatus::SUCCEEDED);
+}
 
-  moveit_msgs::RobotTrajectory traj1;
-  traj1.joint_trajectory.joint_names.push_back("rj1");
-  traj1.joint_trajectory.points.resize(1);
-  traj1.joint_trajectory.points[0].positions.push_back(0.0);
-  if (!tem.push(traj1))
-    ROS_ERROR("Fail!");
-
-  moveit_msgs::RobotTrajectory traj2 = traj1;
-  traj2.joint_trajectory.joint_names.push_back("lj2");
-  traj2.joint_trajectory.points[0].positions.push_back(1.0);
-  traj2.multi_dof_joint_trajectory.joint_names.push_back("basej");
-  traj2.multi_dof_joint_trajectory.points.resize(1);
-  traj2.multi_dof_joint_trajectory.points[0].transforms.resize(1);
-
-  if (!tem.push(traj2))
-    ROS_ERROR("Fail!");
+TEST_F(MoveItCppTest, pushTrajectoryTest)
+{
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj1));
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj2));
+  traj1.multi_dof_joint_trajectory = traj2.multi_dof_joint_trajectory;
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj1));
+}
 
+TEST_F(MoveItCppTest, PushExecuteAndWaitTest)
+{
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj1));
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj2));
   traj1.multi_dof_joint_trajectory = traj2.multi_dof_joint_trajectory;
-  if (!tem.push(traj1))
-    ROS_ERROR("Fail!");
+  ASSERT_TRUE(trajectory_execution_manager_ptr->push(traj1));
+  auto last_execution_status = trajectory_execution_manager_ptr->executeAndWait();
+  ASSERT_TRUE(last_execution_status == moveit_controller_manager::ExecutionStatus::SUCCEEDED);
+}
+
+}  // namespace moveit_cpp
 
-  if (!tem.executeAndWait())
-    ROS_ERROR("Fail!");
+int main(int argc, char** argv)
+{
+  // if (ros::console::set_logger_level(ROSCONSOLE_DEFAULT_NAME, ros::console::levels::Debug))
+  // {
+  //   ros::console::notifyLoggerLevelsChanged();
+  // }
```

Comment:
Please drop this. The debug level can be easily set via a `ROSCONSOLE_CONFIG_FILE`.

Reply:
okay